### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by PepijnSenders**
I fixed some documentation and dead code in the repository.

### CODE REVIEW

File: src/Entity/Tradeability/Shipment.php [hunk 1]
```diff
@@ -83,11 +83,6 @@ public function toNode(DOMDocument $document = null)
 
         $node = $document->createElement('Shipment');
 
-        // Objects first
-        if ($this->getAdditionalInsurance() instanceof AdditionalInsurance) {
```
Comment (stefandoorn) [inline src/Entity/Tradeability/Shipment.php]: Why is this one removed?

File: src/Entity/Tradeability/Shipment.php [hunk 2]
```diff
@@ -335,7 +331,7 @@ public function getProducts()
 
     /**
      * @param array $products
-     * @return QueryRequestShipment
```
Comment (stefandoorn) [inline src/Entity/Tradeability/Shipment.php]: Is returning $this better than QueryRequestShipment? I don't know how this affects auto complete in IDE's. Could you help me what's the best choice?

File: src/TimeInTransit.php [hunk 3]
```diff
@@ -76,15 +76,6 @@ private function sendRequest(TimeInTransitRequest $timeInTransitRequest)
         } else {
             return $this->formatResponse($response);
         }
-
```
Comment (stefandoorn) [inline src/TimeInTransit.php]: Unreachable code?

File: src/Entity/Tradeability/Shipment.php (hunk 2 above)
Comment (PepijnSenders) [inline src/Entity/Tradeability/Shipment.php]: $this is definitely the best choice :) It's best practice.

File: src/TimeInTransit.php (hunk 3 above)
Comment (PepijnSenders) [inline src/TimeInTransit.php]: Yes

**Comment by stefandoorn** [comment]
Hi @PepijnSenders! Great :) I got some shorts questions stated above. If all is clear, I will merge it and put a new release. Could you update the changelog file with  0.7.4 update?

File: src/Entity/Tradeability/Shipment.php (hunk 2 above)
Comment (stefandoorn) [inline src/Entity/Tradeability/Shipment.php]: Great, thanks :)

File: src/Entity/Tradeability/Shipment.php (hunk 1 above)
Comment (PepijnSenders) [inline src/Entity/Tradeability/Shipment.php]: It should never happen, because AdditionalInsurance does not exist.

Comment (stefandoorn) [inline src/Entity/Tradeability/Shipment.php]: Ah, yes. Can you also remove the property + getter/setter? I think it was work in progress that got forgotton, if that class does not exist.

Comment (PepijnSenders) [inline src/Entity/Tradeability/Shipment.php]: Well the getter/setter could stay there, it's just never an instance of `AdditionalInsurance`.

**Pull request merged**